### DISCUSSION TITLE
$table to $column in migration blueprint

### DISCUSSION THREAD

**Discussion by hirenkeraliya** [laravel/framework#42386] [Ideas]
Hey, As we know laravel believes in a readable code. But, on the migration, we wrote like `$table->string('name')` table string name but the table is never `string` we all know that. instead of this, we can do like `$column->string('name')`

```
Schema::create('users', function (Blueprint $column) {
    $column->id();
    $column->string('name');
    $column->timestamps();
});
```

What do you think about this? Is this a good suggestion?

Thanks.

**Comment by adityadees** [comment]
> Hey, As we know laravel believes in a readable code. But, on the migration, we wrote like `$table->string('name')` table string name but the table is never `string` we all know that. instead of this, we can do like `$column->string('name')`
> 
> ```
> Schema::create('users', function (Blueprint $column) {
>     $column->id();
>     $column->string('name');
>     $column->timestamps();
> });
> ```
> 
> What do you think about this? Is this a good suggestion?
> 
> Thanks.

I don't know, but I prefer this way because it's clearly legible.
`Schema::create('users', function (Blueprint $table) = create users table`

rather than
`Schema::create('users', function (Blueprint $column) = create users column`
